src/main/resources/db/migration/all/20230104163132__add_pdu_column_to_premises.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE temporary_accommodation_premises ADD COLUMN pdu TEXT;
+
+-- Some TA premises may have existing rows on the table.
+UPDATE temporary_accommodation_premises
+SET pdu = 'Not specified';
+
+ALTER TABLE temporary_accommodation_premises ALTER COLUMN pdu SET NOT NULL;
+
+-- Some TA premises do not have existing rows on the table, and will need them.
+INSERT INTO temporary_accommodation_premises(premises_id, pdu)
```

Comment:
Will this cause a duplicate PK failure if there are some entries already in temporary_accommodation_premises?

Reply:
I've updated the WHERE clause to:
```sql
WHERE premises.service = 'temporary-accommodation'
AND premises.id NOT IN (
    SELECT premises_id
    FROM temporary_accommodation_premises
)
```
So this should no longer be an issue.